docs/models-definition.md

```diff
@@ -248,13 +241,9 @@ receive:
 
 ```js
 // stored value: ("2016-01-01 00:00:00+00:00", "2016-02-01 00:00:00+00:00"]
-range // [Date, Date]
-range.inclusive // [false, true]
+range // [{ value: Date, inclusive: false }, { value: Date, inclusive: true }]
 ```
 
-Make sure you turn that into a serializable format before serialization since array
```

Comment:
A note here saying that you will need to call `reload` after updating an instance with range type or use `returning: true` option

Reply:
Done.